### PR TITLE
Fix updio join

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-cmgg/germline/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -544,7 +544,7 @@ process {
 
         if(params.updio) {
             withName: "^.*VCF_UPD_UPDIO:UPDIO\$" {
-                ext.prefix = {"updio_${meta.caller}"}
+                ext.prefix = {"updio_${meta.child}_${meta.caller}"}
```
Comment (matthdsm) [inline conf/modules.config:547]: Why the child ID?

**Review by matthdsm — APPROVED** [review]

File: conf/modules.config (hunk 1 above)
Comment (nvnieuwk) [reply conf/modules.config:547]: Because UPDio is run for each child. If a family has multiple children the older ones would get overwritten.

**Review by ToonRosseel — APPROVED** [review]
looking good

File: conf/modules.config (hunk 1 above)
Comment (nvnieuwk) [reply conf/modules.config]: I updated this so one folder will be created per child (a bit similar to how automap outputs its results)

**Pull request merged**